models/tracker/src/migration.ts

```diff
@@ -151,16 +149,13 @@ async function fixProjectsIssueStatusesOrder (tx: TxOperations): Promise<void> {
   await Promise.all(projects.map((project) => fixProjectIssueStatusesOrder(tx, project)))
 }
 
-async function upgradeProjectSettings (tx: TxOperations): Promise<void> {
-  const projects = await tx.findAll(tracker.class.Project, {
-    defaultTimeReportDay: { $exists: false }
-  })
-  await Promise.all(
-    projects.map((project) =>
-      tx.update(project, {
-        defaultTimeReportDay: TimeReportDayType.PreviousWorkDay
-      })
-    )
+async function removeProjectDefaultTimeReportDay (client: MigrationClient): Promise<void> {
```

Comment:
Let's just don't do anything.

Reply:
We do not need to cleanup from existing projects.

Reply:
Done.